bin/plugin/commands/performance.js

```diff
@@ -344,7 +356,12 @@ async function runPerformanceTests( branches, options ) {
 	// 4- Running the tests.
 	log( '\n>> Running the tests' );
 
-	const testSuites = [ 'post-editor', 'site-editor' ];
+	const testSuites = [
+		'post-editor',
+		'site-editor',
+		'front-end-twentytwentyone',
+		'front-end-twentytwentythree',
```

Comment:
I wonder if it makes sense to change these names into more semantic ones. Like `front-end-classic-theme` and `front-end-block-theme`?